Fullstacker/DataLayer/Repositories/InventoryRepository.cs

```diff
@@ -0,0 +1,52 @@
+﻿using DataLayer.Contexts;
+using DataLayer.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataLayer.Repositories
+{
+    public class InventoryRepository : IInventoryRepository
+    {
+        private readonly DataContext _dataContext;
+
+        public InventoryRepository(DataContext dataContext)
+        {
+            _dataContext = dataContext;
+        }
+
+        public async Task<IList<Entities.Inventory>> Get()
+        => await _dataContext.Inventory.Include(x => x.Item).ToListAsync();
```

Comment:
both Get methods here contains more fields than were required in the todo. E.g. ItemId, etc. Need to filter more